participant/src/args.rs

```diff
@@ -0,0 +1,19 @@
+use clap::Parser;
+
+#[derive(Parser, Debug, Default)]
+#[command(author, version, about, long_about = None)]
+pub struct Args {
+    /// Public key package to use. Can be a file with a JSON-encoded
+    /// package, or "-". If the file does not exist or if "-" is specified,
+    /// then it will be read from standard input.
+    #[arg(short = 'P', long, default_value = "key-package-1.json")]
```

Comment:
I think we should change the short name, I'm changing it here to match #132 

```suggestion
    #[arg(short = 'k', long, default_value = "key-package-1.json")]
```